doc/nrf/config_and_build/sysbuild_migration.rst

```diff
@@ -0,0 +1,579 @@
+| :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_CUSTOM`     | Sign b0 images with a custom command                                                    |
++------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_KEY_FILE`   | Absolute path to signing private key file                                               |
++------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_COMMAND`    | Command called for custom signing, will have file to sign provided as an argument       |
++------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_PUBLIC_KEY` | Absolute path to signing key public file                                                |
++------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_SECURE_BOOT_PUBLIC_KEY_FILES`   | Comma-separated value list of absolute paths to signing public key files                |
++------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+
+Secure boot now allows for being enabled centrally from sysbuild for both application and network cores for nRF53-based boards. Configuration that was done in images themselves before must now be applied at a sysbuild level, if not then the secure boot image(s) will not be built or settings of these builds will be forcefully replaced with the default which sysbuild generates, which will make firmware updates incompatible with firmware images built in previous versions of |NCS|.
+
+.. _child_parent_to_sysbuild_migration_bluetooth_fast_pair:
+
+Bluetooth fast pair
+###################
+
+Bluetooth fast pair hex generation is now handled by sysbuild, the following Kconfig options exist:
+
++------------------------------------------+-------------------------------------------+
+| Kconfig option                           | Description                               |
++==========================================+===========================================+
+| :kconfig:option:`SB_CONFIG_BT_FAST_PAIR` | Enables Bluetooth fast pair functionality |
++------------------------------------------+-------------------------------------------+
+
+Applications must set this at a sysbuild level if they want the Bluetooth fast pair data generated, the method of supply the key data via command line remains unchanged as it worked in previous |NCS| versions.
```

Comment:
```suggestion
Applications must set this at the Sysbuild level if you want the Bluetooth Fast Pair data generated.
The method of supplying the key data via the command line remains unchanged from previous |NCS| versions.
```

Reply:
I think we should link to the docs explaining how to supply the key data via CLI

Reply:
@MarekPieta do you have a link?